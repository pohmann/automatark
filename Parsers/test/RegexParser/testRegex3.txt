^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$
a$
^a$
^a
aaaa
bob
a|b
abc
[abc]
^(0|(([1-9]{1}|[1-9]{1}[0-9]{1}|[1-9]{1}[0-9]{2}){1}(\ [0-9]{3}){0,})),(([0-9]{2})|\-\-)([\ ]{1})(???|EUR|EURO){1}$