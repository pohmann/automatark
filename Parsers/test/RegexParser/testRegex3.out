SOS(([Char:a-Char:z Char:A-Char:Z Char:0-Char:9 Char:_ Char:. Char:+ Char:- ])+)(Char:@)(([Char:a-Char:z Char:A-Char:Z Char:0-Char:9 Char:- ])+)(Escaped:.)(([Char:a-Char:z Char:A-Char:Z Char:0-Char:9 Char:- Char:. ])+)EOS
(Char:a)EOS
SOS(Char:a)EOS
SOS(Char:a)
(Char:a)(Char:a)(Char:a)(Char:a)
(Char:b)(Char:o)(Char:b)
((Char:a) | (Char:b))
(Char:a)(Char:b)(Char:c)
([Char:a Char:b Char:c ])
